cli/src/platform-worker.ts

```diff
@@ -59,7 +64,7 @@ export class PlatformWorker {
           // Trigger the identifyHandler with scan results.
           await this.mockLocalHomePlatform.triggerIdentify(
             scanMessage.requestId,
-            scanRemoteInfo.buffer,
+            Buffer.from(scanRemoteInfo.scanData, 'hex'),
```

Comment:
what about doing `hex`/`Buffer` related bug fixes in a separate change?

Reply:
Will do.

Reply:
Done. see #59